pkg/libvirttools/virtualization.go

```diff
@@ -1096,13 +1096,13 @@ func (v *VirtualizationTool) UpdateDomainResources(vmID string, lcr *kubeapi.Lin
 	var memUpdated bool
 
 	// TODO: enable this after CPU bug fix
```

Comment:
Need to remove this comment line as well.

Reply:
removed. thanks for catching this up.